packages/test-interface-criteria/src/criteria/CriteriaFutureScheduledOpportunity.js

```diff
@@ -12,7 +12,7 @@ module.exports = class CriteriaFutureScheduledOpportunity extends Criteria {
     var eventStatus = opportunity.eventStatus;
 
     if (
-      !(Date.parse(startDate) > new Date(Date.now() + (3600 * 1000 * 2)))
+      !(Date.parse(startDate) > (new Date(Date.now() + (3600 * 1000 * 2))).getTime())
```

Comment:
![Screenshot 2020-08-14 at 11 36 52](https://user-images.githubusercontent.com/2067438/90241611-5638b500-de23-11ea-8780-2f4ee46c44f1.png)

TypeScript noted this as an error. It is technically possible to compare a JS date and an integer, but it is generally recommended against doing this sort of thing (as is e.g. comparing "2" with 2 - something else that JS allows but TS does not). This might be because TS always turns on JS strict mode 🤔

Reply:
Good catch!